packages/contentstack-import/src/import/modules-js/entries.js

```diff
@@ -486,7 +498,23 @@ module.exports = class ImportEntries {
           return resolve();
         })
         .catch((error) => {
-          addlogs(this.config, chalk.red("Failed to create entries in '" + lang + "' language"), 'error');
+          let title = "";
+          if(error?.request?.data){
+            if(JSON.parse(error.request.data).entry.title!==undefined) {
```

Comment:
we need a better logic here to avoid any issues

Reply:
@shafeeqd959  made the changes please review it